tests/org.eclipse.xtend.core.tests/src/org/eclipse/xtend/core/tests/validation/ValidationBug427432Test.xtend

```diff
@@ -31,7 +31,7 @@ class ValidationBug427432Test extends AbstractXtendTestCase {
 		'''
 			class Foo {
 			
-				val x = 1
+				static val x = 1
```

Comment:
The same applies to other test cases: `finalInstanceField_2()`, `finalInstanceField_3()`, `notFinalInstanceField()`, `notFinalStaticField()`, `notFinalStaticField_2()`, etc.
If we want the dead code to be detected in all these cases, we have to change the implementation of that validation. But that would be worth another bug IMO.